test/Recall.lean

```diff
@@ -0,0 +1,89 @@
+import Std.Tactic.GuardMsgs
+import Mathlib.Tactic.Recall
+import Mathlib.Analysis.Calculus.Deriv.Basic
+import Mathlib.Analysis.SpecialFunctions.Trigonometric.Basic
+import Mathlib.Data.Complex.Exponential
+
+-- Remark: When the test is run by make/CI, this option is not set, so we set it here.
+set_option pp.unicode.fun true
```

Comment:
This is a comment about tests in general rather than just this PR: I wonder if we could have the tests use the same options as mathlib? I had to set this option too in the tests for [`#explode`](https://github.com/leanprover-community/mathlib4/blob/master/test/Explode.lean#L27).